### PR TITLE
Quick Fix

### PR DESCRIPTION
**Pull request by jacklul**
Missed this in #188, sorry!

### CODE REVIEW

File: src/Entities/MessageEntity.php [hunk 1]
```diff
@@ -18,23 +18,22 @@ class MessageEntity extends Entity
...
     {
         $this->type = isset($data['type']) ? $data['type'] : null;
-        if (empty($this->type)) {                               //
+        if (empty($this->type)) {
             throw new TelegramException('type is empty!');
         }
 
         $this->offset = isset($data['offset']) ? $data['offset'] : null;
-        if (empty($this->offset) && $this->offset != 0) {       //
+        if ($this->offset === '') {
```
Comment (noplanman) [inline src/Entities/MessageEntity.php]: I haven't checked, but you're sure that `$this->offset` can't be `0`?

Comment (jacklul) [inline src/Entities/MessageEntity.php]: Straight from logs from one of my bots:
```
"entities":[{"type":"bot_command","offset":0,"length":7}]
"entities":[{"type":"bold","offset":350,"length":0}]
```

Comment (noplanman) [inline src/Entities/MessageEntity.php]: Ok, so it can be `0`. And clearly you've tested this, yes? 😊 

Sorry, I'm sometimes a bit paranoid about changes like this, as they sometimes might introduce the little edge cases that produce weird errors.

Comment (jacklul) [inline src/Entities/MessageEntity.php]: Yep, tested!

**Comment by MBoretto** [comment]
We can create a test for this
